webui/src/pages/repositories/repository/changes.jsx

```diff
@@ -258,7 +258,6 @@ const ChangesBrowser = ({repo, reference, prefix, onSelectRef, }) => {
                         <span className="float-start">
                             {(delimiter !== "") && (
                                 <URINavigator path={prefix} reference={reference} repo={repo}
-                                              relativeTo={`${reference.id} workspace`}
```

Comment:
You sure we want to remove the `rerference.id` too?

Reply:
relativeTo is not needed, once it is empty the URI nevigator provides the lakefs URI using the reference id as in the Objects view